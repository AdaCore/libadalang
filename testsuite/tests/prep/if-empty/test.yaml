driver: prep
configs:
    input.ada: ["-DX"]
