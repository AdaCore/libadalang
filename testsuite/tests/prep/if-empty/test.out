== input.ada ==

# -DX

with Bar;

procedure Foo is
begin
   Bar (1);
end Foo;

